exec/java-exec/src/test/java/org/apache/drill/test/TestGracefulShutdown.java

```diff
@@ -163,7 +163,7 @@ public void run() {
         }
       }).start();
 
-      Thread.sleep(grace_period);
+      Thread.sleep(gracePeriod);
```

Comment:
I don't think we need this sleep here since we are already checking the termination condition in the loop below.